finalPrep/README.md

```diff
@@ -32,23 +32,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![Screen Shot 2021-12-17 at 6 05 59 PM](https://user-images.githubusercontent.com/3982238/146623794-2558406b-12ea-478d-a5ec-e7ac0cc5d047.png)
+
 - `Add Week 2 Screenshot Here`
+- ![Screen Shot 2021-12-17 at 6 07 47 PM](https://user-images.githubusercontent.com/3982238/146623830-46d65340-2c49-4dd4-9a48-28a4703fa888.png)
+
 - `Add Week 3 Screenshot Here`
+- ![Screen Shot 2021-12-17 at 6 08 34 PM](https://user-images.githubusercontent.com/3982238/146623855-b6993aed-db58-4427-92a7-f11bb129a17b.png)
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [LINK](https://gist.github.com/Abumpus1/753e6d1b34b412eb45ed2cca5c78be57)
```

Comment:
Awesome I noticed what you wrote about calendaring - connects to my previous feedback! I think this will be a great start for you and your mentors working relationship.